cloud-recording/develop/integration-best-practices.md

```diff
@@ -80,14 +81,48 @@ For example, if there is a fixed 5-minute break in each class, you can set `maxI
-
-For scenes with a large audience in the channel or high availability requirements, consider whether you can 
-accept the impact of high availability migration based on your own business characteristics, and decide whether to adopt 
-higher quality assurance measures. For example, create multiple recording tasks for critical scenes using different UIDs. 
-Alternatively, you can make periodic API calls and monitor notifications to get the latest recording task status, 
-then create a new cloud recording task with a different UID once you confirm the recording task status is unhealthy.
-
-If you create multiple cloud recording tasks, you are charged separately for each of them. For details, see 
-[Pricing](../reference/pricing).
\ No newline at end of file
+Network failures and potential risks may occur due to factors such as cloud and network software, infrastructure, and other elements outside of <Vg k="COMPANY"/>'s control. To enhance the user experience, Cloud Recording offers automatic high availability task migration for failure recovery. When a failure is detected, the recording task will be migrated within 90 seconds. During this time, the recording may be disrupted and recorded files may be lost.
+
+For scenes with a large audience in the channel or high availability requirements, consider whether you can accept the impact of high availability migration based on your own business characteristics, and decide whether to adopt higher quality assurance measures. For example, create multiple recording tasks for critical scenes using distinct UIDs, and ensure use of servers in different geographical regions through geofencing. Alternatively, you can make periodic API calls and monitor notifications to get the latest recording task status, then create a new cloud recording task with a different UID once you confirm the recording task status is unhealthy.
+
+If you create multiple cloud recording tasks, you are charged separately for each of them. For details, see [Pricing](../reference/pricing).
+
+## Integration requirements checklist
+
+To ensure reliability of the cloud recording service, refer to the following checklist to confirm that your solution meets the integration requirements:
+
+| Serial | Importance | Item | Description |
+|--|----------|----------|----------|
+| 1 | required | Subscribe to a service | Make sure you have activated the cloud recording service. | 
+| 2 | required | request method | <ul><li>To query, use the `POST` request method; to query the recording status use `GET`. </li> <li>Request URLs and request body content are case-sensitive.</li></ul> |
+| 3 | required | Get recording resources | <ul><li>The passed in `uid` cannot duplicate any UID within the current channel.</li><li>For page recording, make sure an appid + cname + uid corresponds to a resource ID.</li><li>Make sure that a resource ID is only used for one cloud recording service.</li><li>Make sure to call the `start` method.</li></ul> |
```

Comment:
I'd say appid + cname + uid  should be in code font

Reply:
fixed